help/search/_posts/2016-07-29-comprehensive-solr-term-list.md

```diff
@@ -13,112 +13,126 @@ It is a technical document and you probably don't need to read it unless you are
+ |                 classic_factor | y | y | Integer values containing the boost factor used by ADS Classic. In essence log(1 + cites + norm_reads) where number of citations has been normalized and the whole value is multiplied by 5000 and then cast to Integer. | 
+ |                        comment | y | y | Kitchen sink for holding various bits of information not available elsewhere (probably only useful if you are curating ADS records) | 
+ |                      copyright | y | y | Copyright given by the publisher | 
+ |                           data | y | y | List of sources that hold data associated with this paper (record) - format is `name:count`, i.e. `Chandra:3` | 
+ |                     data_facet | y | n | Data sources for the paper (without counts, but the counts can be retrieved when faceting on the values of this field) | 
+ |                       database | y | y | Database (collection) into which the paper was classified, a paper can belong to more than one| 
+ |                           date | y | y | Machine readable version of `pubdate`, time format: `YYYY-MM-DD'T'hh:mm:ss.SSS'Z'`| 
+ |                        doctype | y | y | Types of document: abstract, article, book, bookreview, catalog, circular, editorial, eprint, erratum, inbook, inproceedings, mastersthesis, misc, newsletter, obituary, phdthesis, pressrelease, proceedings, proposal, software, talk, techreport| 
+ |             doctype_facet_hier | y | n | Hierarchical facets consisting of nested document types | 
+ |                            doi | y | y | Digital object identifier | 
+ |                         editor | y | y | Typically for books or series, similar rules to `book_author` | 
+ |                            eid | y | y | Electronic id of the paper (equivalent of page number) | 
+ |                          email | y | n | List of e-mails for the authors that included them in the article (is only accessible to users with elevated privileges) | 
+ |                        entdate | y | y | Creation date of ADS record in user-friendly format (YYYY-MM-DD) | 
+ |                     entry_date | y | y | Creation date of ADS record in RFC 3339 (machine-readable) format | 
+ |                       esources | y | y | Types of electronic sources available for a record (e.g. pub_html, eprint_pdf) | 
+ |                       facility | y | y | List of facilities declared in paper (low count field for now) | 
+ |                   first_author | y | y | First author of the paper | 
+ |        first_author_facet_hier | y | n | See `author_facet_hier` | 
+ |              first_author_norm | y | n | See `author_norm` | 
+ |                 fulltext_mtime | y | y | Machine readable modification timestamp; corresponds to time when a fulltext was updated | 
+ |                          grant | y | y | Field that contains both grant ids and grant agencies. | 
+ |                 grant_agencies | y | y | Index with just the grant agencies names (e.g. NASA). | 
```

Comment:
I assume you talk about **grant_agencies** -- I can remove it from the help - also `grant_id`

however, `grant` right now contains ['agancy', 'grant_id'] -- it would be more logical to start indexing grant_id into a separate field, ditto for agency; and make `grant` to be a virtual field

should we do that?

Reply:
No, see my comment above about the need to move away from grant in favor of `award`.  But until we have a plan for this (and a data model for the award use cases) we should hold off.